src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml

```diff
@@ -3514,48 +3590,51 @@ components:
             An optional transaction message. Only affects the costing.
             This type is defined in the Core API as `TransactionMessage`. See the Core API documentation for more details.
         flags:
-          type: object
-          required:
-            - use_free_credit
-            - assume_all_signature_proofs
-            - skip_epoch_check
-          properties:
-            use_free_credit:
-              description: |
-                Whether to use a virtual, preview-only pool of XRD to pay for all execution fees.
-              type: boolean
-            assume_all_signature_proofs:
-              description: |
-                Whether the virtual signature proofs should be automatically placed in the auth zone.
-              type: boolean
-            skip_epoch_check:
-              description: |
-                Whether to skip the epoch range check (i.e. ignoring the `start_epoch_inclusive` and
-                `end_epoch_exclusive` parameters, if specified).
+          $ref: "#/components/schemas/TransactionPreviewFlags"
```

Comment:
Possibly just `PreviewFlags` as they might apply to other kinds of preview (e.g. Manifest preview or Subintent preview if we add these)

Reply:
Also, could we make "flags" non-required, as per the node? :)